app/models/ability.rb

```diff
@@ -0,0 +1,17 @@
+ class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new # guest user
+    can :read, :all
+    can [:update, :destroy, :create], [Comment]
```

Comment:
Wouldn't this allow any user to update and destroy comments for anyone? Also, I didn't find any update action in comments controller.